### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by nini-faroux**
Add some more tests

### CODE REVIEW

**Comment by nini-faroux** [comment]
@Randy1Burrell just realised I hadn't taken this off draft, set it as ready for review now

File: triphut-specs/Spec/Vote/ContextValidator.hs [hunk 1]
```diff
@@ -18,33 +30,119 @@ import Plutus.Model (
...
+ )
 import Spec.Tally.Utils (findTally, findTallyConfig)
 import Spec.Values (dummyTallyValue, dummyVoteValue)
 import Spec.Vote.Script (voteTypedValidator)
-import Spec.Vote.Transactions (runInitVote, runInitVoteConfig)
+import Spec.Vote.Transactions (runInitVote, runInitVoteConfig, runInitVoteMinterConfig, runInitVoteWithUser)
 import Spec.Vote.Utils (findVote, findVoteConfig)
-import Triphut.Vote (VoteActionRedeemer (Count))
-import Prelude (mconcat, pure, show, (*), (+), (<>))
+import Triphut.Vote (VoteActionRedeemer (Cancel, Count))
```
Comment (Randy1Burrell) [inline triphut-specs/Spec/Vote/ContextValidator.hs]: This should be changes to `dao`

**Review by Randy1Burrell — CHANGES_REQUESTED** [review]
@nini-faroux I left a comment.

File: triphut-specs/Spec/Vote/ContextValidator.hs (hunk 1 above)
Comment (nini-faroux) [reply triphut-specs/Spec/Vote/ContextValidator.hs]: They're changed to dao everywhere now @Randy1Burrell

**Review by beezzy1984 — APPROVED** [review]
@nini-faroux LGTM! 💯

**Review by Randy1Burrell — APPROVED** [review]
@nini-faroux LGTM! 🥇

**Comment by nini-faroux** [comment]
Thanks @Randy1Burrell !

**Pull request merged**